pkg/api/arc.yml

```diff
@@ -551,21 +538,8 @@ components:
     TransactionSubmitStatus:
```

Comment:
I think you can now remove the whole `TransactionSubmitStatus` schema as it does not contain anything anymore